digitalocean/table_digitalocean_database.go

```diff
@@ -118,6 +119,63 @@ func getDatabase(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData
 	return *result, nil
 }
 
+func getUsers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func getDatabaseUsers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```
Also change the same in log statement.